src/main/io/osd.c

```diff
@@ -600,6 +601,27 @@ static bool osdDrawSingleElement(uint8_t item)
 
             break;
         }
+		
+    case OSD_MOTOR_DIAG:
+    {
+        if( areMotorsRunning() ) {
+            int motorDiag[getMotorCount()];
+            int maxMotorPctValue = -1;
```

Comment:
```
float sum = 0;
int maxIdx = 0;
for (int i = 0; i < getMotorCount(); i++) {
  sum += motor[i];
  if (motor[i] > motor[maxIdx]) {
    maxIdx = i;
  }
 }
 tfp_sprintf(buff, "MAX M%d=%3d AVG=%3d", maxIdx, fScaleRange(motor[maxIdx], motorOutputLow, motorOutputHigh, 0, 100),  fScaleRange(sum / getMotorCount(), motorOutputLow, motorOutputHigh, 0, 100));
```
`fScaleRange` is not in BF code, but can be copied  as float version of `scaleRange`

Reply:
Yes ... it may eventually cause problems if motorOutputLow -  motorOutputHigh gets small (0-1 range for motor), but for now it's good enough ... 

BTW: conversion to int is missing in my code